okhttp-tests/src/test/java/okhttp3/HeadersTest.java

```diff
@@ -366,8 +370,14 @@
         .build();
     List<Challenge> challenges = HttpHeaders.parseChallenges(headers, "WWW-Authenticate");
     assertEquals(1, challenges.size());
-    assertEquals("Digest", challenges.get(0).scheme());
+    assertEquals("digest", challenges.get(0).scheme());
     assertEquals("myrealm", challenges.get(0).realm());
+    Map<String, String> expectedAuthParams = new HashMap<>();
```

Comment:
nit: we never use HashMap and you shouldn’t either!
https://publicobject.com/2016/02/08/linkedhashmap-is-always-better-than-hashmap/

Reply:
Well, it's only in the tests, but sure, I'll replace it, no problem